pkg/apis/extensions/types.go

```diff
@@ -676,6 +676,12 @@ type PodSecurityPolicySpec struct {
 	// If set to false the container may run with a read only root file system if it wishes but it
 	// will not be forced to.
 	ReadOnlyRootFilesystem bool `json:"readOnlyRootFilesystem,omitempty"`
+	// SeccompProfiles lists the allowed profiles that may be set for the pod or
+	// container's seccomp annotations.  An unset (nil) or empty value means that no profiles may
+	// be specifid by the pod or container.	The wildcard '*' may be used to allow all profiles.  When
+	// used to generate a value for a pod the first non-wildcard profile will be used as
```

Comment:
> For item 2 - this is the same as disallowing all profiles (IMO). If you disallow all profiles then you are requesting the cluster to act in a sane way on your behalf.

And in the case where you'd like to dictate what "sane" behavior is then you allow profiles and use the defaulting to provide the profile that you consider to be the best choice for anyone not requesting a specific profile.